### PR TITLE
 Set and read secure cookies

### LINKED ISSUE #11: Set the user's cookie on login
Something @kwypchlo brought up today was that I can set the user's cookie right here instead of relying on the FE to do that for me. I'll do that in a follow up.

_Originally posted by @ro-tex in https://github.com/NebulousLabs/skynet-accounts/pull/8#r532747939_

### PR DESCRIPTION
**Pull request by ro-tex**
Closes #11

### CODE REVIEW

File: api/cookie.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+const (
+	// cookieName is the name of the cookie where we store the user's JWT token.
+	cookieName = "SkynetJWTCookie"
+	// cookieValidity specifies for how long the cookie is valid. In seconds.
+	cookieValidity = 604800 // one week
+
+	// envCookieDomain holds the name of the environment variable for the
+	// domain name of the portal
+	envCookieDomain = "COOKIE_COMAIN"
```
Comment (m-cat) [inline api/cookie.go:19]:
```suggestion
	envCookieDomain = "COOKIE_DOMAIN"
```

Comment (m-cat) [reply api/cookie.go:19]: Why have env vars to store the names of other env vars? I feel like we have a lot of env vars and it's getting confusing

**Review by m-cat — CHANGES_REQUESTED** [review]
Just 1 comment

File: api/cookie.go (hunk 1 above)
Comment (ro-tex) [reply api/cookie.go:19]: Ha ha ha, Clippy!

**Review by m-cat — APPROVED** [review]

**Comment by ro-tex** [comment]
The changes in this PR are no longer needed, considering the move to Kratos. Closing.

**Pull request closed without merging**